packages/react-composites/src/composites/localization/locales/zh-CN/strings.json

```diff
@@ -165,8 +165,8 @@
     "unableToResolveTenantMoreDetails": "请检查正确输入参与者 ID",
     "participantIdIsMalformedTitle": "参与者 ID 的格式不正确",
     "participantIdIsMalformedMoreDetails": "请检查参与者 ID 的格式正确",
-    "moreButtonGalleryControlLabel": "Gallery options",
-    "moreButtonGalleryPositionToggleLabel": "Move gallery to top"
+    "moreButtonGalleryControlLabel": "库选项",

```

Comment:
Not related to this PR, this Simplified Chinese translation looks a bit strange while traditional Chinese looks more correct

@mgamis-msft do you know where I can send a request to correct this?

Reply:
Sorry just saw this. But there are two ways documented here https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/25949/Localization?anchor=filing-an-issue-with-translated-strings